var/spack/repos/builtin/packages/py-icselector/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyIcselector(PythonPackage):
+    """Python library for ion channel model selection."""
+
+    homepage = "https://bbpgitlab.epfl.ch/msg/icselector"
+    git = "ssh://git@bbpgitlab.epfl.ch/msg/icselector.git"
+
+    version("0.0.3")
```

Comment:
This needs a tag.

Reply:
Ok thanks, I'll ask the package author to create one then

Reply:
@vandorp we're in the process of adding a package that depends on Icselector, and I noticed that you're one of the authors of this package. Could you please create a tagged version for it on Gitlab?

Reply:
Don't forget to add it to the `version` statement, that should take care of the `Error: Package PyIcselector has no version with a URL` issue as well.

Reply:
Yes, I took care of it in the last commit